<launch>

    <arg name="global_frame"    default="map"/>
    <arg name="base_frame"      default="base_link"/>
    <arg name="pub_topic"       default="/amcl_pose"/>
    <arg name="sub_topic"       default="/ndt_pose"/>

    <node pkg="pose_filter" name="pose_filter" type="pose_filter_node" output="screen">
        <param name="global_frame"  value="$(arg global_frame)"/>
        <param name="base_frame"    value="$(arg base_frame)"/>
        <param name="pub_topic"     value="$(arg pub_topic)"/>
        <param name="sub_topic"     value="$(arg sub_topic)"/>
    </node>

</launch>